fairworkflows/rdf_wrapper.py

```diff
@@ -164,15 +164,23 @@ def from_nanopub(cls, uri: str, use_test_server=False):
         self._is_published = True
         return self
 
-    def _publish_as_nanopub(self, use_test_server=False):
+    def _publish_as_nanopub(self, use_test_server=False, **kwargs):
```

Comment:
I hesitated a bit with this, the alternative would be to 'hardcode' all arguments for `nanopub.Publication.from_assertion()` in fairworkflows (in `_publish_as_nanopub` and for `FairStep.publish_as_nanopub` and `FairWorkflow.publish_as_nanopub`. I prefer this, because:
* more flexible
* simpler code

But I am curious what the reviewer thinks!

Reply:
Honestly, I strongly prefer this way of doing it. I don't think much would be gained by hardcoding all the kwargs explicitly except for breaking every time nanopub is updated. If a specific downside comes up then we'll deal with it then, but for now this looks great to me.

Reply:
The ```invalid_kwarg``` and ```_merge_derived_from()``` code you added solves any issues I would have foreseen with blindly passing kwargs, so this actually looks fairly robust.

Reply:
Cool :)